app/app.rb

```diff
@@ -2,12 +2,27 @@
 
 require 'sinatra'
 require_relative 'database_setup'
+require_relative './models/property'
 
 class MakersBNB < Sinatra::Base
 
 get '/' do
   'Welcome!'
 end
 
+get '/property' do
+  @property = Property.all
+  erb :'links/property'
+end
+
+post '/property' do
+  property = Property.create(name: params[:name])
```

Comment:
This all looks good, but we have named our params/form names with the first letter capitalised. For consistency let's either have these as all small letters or with first letter capitalised. Let's discuss.